LadyBug/Modules/Posts/PostDetails Module/Presenter/PostDetailsPresenter.swift

```diff
@@ -125,4 +127,21 @@ class PostDetailsPresenter: PostDetailsPresenterProtocol {
             }
         }
     }
+
+    func createComment(content: String, postId: Int) {
```

Comment:
rename this to AddComment and make take image from view controller and then make the image == nil after success